railgun/test/integration/ingress_https_test.go

```diff
@@ -355,6 +355,38 @@ func TestHTTPSIngress(t *testing.T) {
 		}
 	}()
 
+	t.Logf("checking ingress %s status readiness.", ingress1.Name)
+	assert.Eventually(t, func() bool {
```

Comment:
It would appear that we may want to `require` instead of `assert` here, because if the status doesn't resolve there's not really anything further we can do right?
```suggestion
	require.Eventually(t, func() bool {
```